builtin/settingtypes.txt

```diff
@@ -522,6 +522,18 @@ console_color (Console color) string (0,0,0)
 #    In-game chat console background alpha (opaqueness, between 0 and 255).
 console_alpha (Console alpha) int 200 0 255
 
+#    Formspec full-screen background opacity (between 0 and 255).
+formspec_fullscreen_bg_opacity (Formspec Full-Screen Background Opacity) int 140 0 255
```

Comment:
`formspec_backdrop_bg_opacity` ?

or maybe mention behind in the description at least